tensorflow/core/kernels/BUILD

```diff
@@ -41,6 +41,8 @@ load(
 load(
     "@local_config_rocm//rocm:build_defs.bzl",
     "if_rocm",
+    "rocm_hipblaslt",
```

Comment:
This is apparently unused, and our BUILD file checker is asking for it to be removed.

Reply:
sorry, overlooked this comment. Now I rebased the branch and fixed this